bp_top/test/tb/bp_top_fpga/bp_fpga.tcl

```diff
@@ -0,0 +1,2599 @@
+#*****************************************************************************************
```

Comment:
Is there something more minimal that we can use to recreate the project? It's just that this is a pretty big file and I'm worried that when we regenerate it, it will create enormous diffs every time.

Reply:
The only time we would like to regenerate it is: